app/views/root/campaign.html.erb

```diff
@@ -1,52 +1,25 @@
 <% content_for :title, page_title(@publication) %>
+<div class="header-context group"><!-- deliberately empty --></div>
+
 <main id="content" role="main" class="group campaign">
   <section id="landing">
-    <h1><%= @publication.title %></h1>
-    <% if @publication.details['organisation'] %>
-    <div class="organisation">
-      <a href="<%= organisation_url(@publication) %>" class="organisation-logo organisation-logo-<%= organisation_crest(@publication) %> <%= organisation_brand_colour(@publication) %>"><%= formatted_organisation_name(@publication) %></a>
+    <div class="campaign-image">
+      <img src="<%= campaign_image_url(@publication, 'large') %>" alt="Banner">
```

Comment:
I don't think this alternative text adds anything useful for assistive technology. Unless we are going to get publishers to add in dynamic alternative text I would be inclined to just not have an `alt` attribute.